iCub/package.xml

```diff
@@ -1,14 +1,14 @@
 <package>
   <name>iCub</name>
-  <version>1.6.0</version>
+  <version>1.18.0</version>
```

Comment:
Are we planning on a release soon? I see the current version is 1.17.1.

Reply:
Yes, we are planning to merge this in master and to the 1.18.0 for the 2020.11 sw distro.